### ISSUE TITLE
Improvements to CSS Props for DataTable

### ISSUE THREAD

**Issue by HardeepAsrani** [Codeinwp/visualizer#450]
There are a few issues with it:

- `transform` property is of no use, and we should remove it. Not sure if anyone uses it.
- We have three CSS areas, which are the following:

<img width="302" alt="Screen Shot 2019-07-16 at 1 42 42 pm" src="https://user-images.githubusercontent.com/2649903/61277394-b9df5180-a7cf-11e9-99db-4c665c1f75cb.png">

Table Cell overrides both Even/Odd columns, and it makes no sense to have it. We should replace it to "Header Row" and use its values to style the Table header instead as we do it Google Chart's Table.

@contactashish13

**Comment by contactashish13** [comment]
@HardeepAsrani transform was part of the old table chart as well, so it has been retained. Why we included it is something we will have to go back in history for. We don't know if someone uses it or not so we can't drop it just because we don't understand its value.  We should retain it to ensure consistency in both interfaces. 

PR https://github.com/Codeinwp/visualizer/pull/454 fixes the issues with row/cell settings. It has been merged into 3.3.0.
